src/ShopBundle/objects/TCMSListManager/TCMSListManagerShopArticles.php

```diff
@@ -25,21 +25,18 @@ protected function PostCreateTableObjectHook()
     {
         parent::PostCreateTableObjectHook();
 
-        $isActive = $this->tableObj->getCustomSearchFieldParameter('is_active');
-
-        if (null === $isActive) {
-            $isActive = '';
-        }
-
-        $filterArticleType = $this->tableObj->getCustomSearchFieldParameter('filterArticleType');
+        $request = $this->getRequest();
 
-        if (null === $filterArticleType) {
-            $filterArticleType = 'all';
+        if ($request->query->has('is_active')) {
+            $isActive = $request->query->get('is_active');
+        } else {
+            $isActive = $this->tableObj->_postData['is_active'] ?? '';
```

Comment:
The value used to be accessed via `getCustomSearchFieldParameter` - why the change to _postData?

Reply:
The custom search field parameters seem to be an (old?) unused mechanism that is only found here.

Other similar locations (e. g. for "searchTerm") are solely using "_postData" (which actually translates to "all data for the current page including stored ones".

With the custom search field parameters used this is a 4-times indirection on the "page data".
With "_postData" it is "only" 2-times indirection.